### PR TITLE
Fixed issue #20305 Update button on payment checkout is not proper alligned

### PR DESCRIPTION
**Pull request by GovindaSharma**
### Description (*)
Fixed issue #20305 Update button on payment checkout is not proper alligned

### Fixed Issues (if relevant)

1. magento/magento2 #20305: Update button on payment checkout is not proper alligned
2. ...

### Manual testing scenarios (*)

   1. Go to frontend
    2.Add product to cart
    3.Goto Checkout
    4.On Payment Information uncheck the box of My billing and shipping address are the same
    5.Go down and check update button

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/checkout/_payments.less [hunk 1]
```diff
@@ -118,6 +118,7 @@
             .primary {
                 .action-update {
                     margin-right: 0;
+                    margin-bottom:20px;
```
Comment (VladimirZaets) [inline app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/checkout/_payments.less]: Hi @GovindaSharma. Thanks for collaboration. Please add indent between value and property declaration.

**Comment by GovindaSharma** [comment]
@VladimirZaets i have fixed the indents please check and approve thanks.

**Comment by VladimirZaets** [comment]
@GovindaSharma thanks!

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3900](https://jira.corp.magento.com/browse/ENGCOM-3900) has been created to process this Pull Request

**Comment by VladimirZaets** [comment]
Hi @GovindaSharma. Thanks for collaboration.
We applied your fix and issue [#20305](https://github.com/magento/magento2/issues/20305) still reproduced

**Comment by GovindaSharma** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma, here is your [new Magento instance](https://pr-20307.instances.magento-community.engineering).
Admin access: [https://pr-20307.instances.magento-community.engineering/admin](https://pr-20307.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by GovindaSharma** [comment]
@VladimirZaets ,thank you for your review,i have checked in test instance it is working fine.Here is the screenshot of it,
screenshot-pr-20307 instances magento-community engineering-2019-01-29-11-14-45

As you can see update button is not too close with phone number,in the issue listed it is too close with phone no field.

Actual issue screenshot is this

51887173-cc4c0700-23b8-11e9-8501-eaae76f61c51

As you can see from above screenshot ,it is too close with phone no field


Please have a look once again.

Thank You

**Comment by VladimirZaets** [comment]
Hi @GovindaSharma. We will retest this PR and leave feedback. Thanks

**Comment by sdzhepa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa, here is your [new Magento instance](https://pr-20307.instances.magento-community.engineering).
Admin access: [https://pr-20307.instances.magento-community.engineering/admin](https://pr-20307.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Pull request merged**